Update N/workbook types

brunogaleski opened the pull request:
Update some types from N/workbook module, mainly related to chart creation. I have these changes being used in a SuiteApp for a while and they work fine.

MrRob commented:
@brunogaleski This is good, I appreciate the updates.  I do have a few questions though:

1. Regarding Chart Definitions - I couldn't find that in the NetSuite help documentation, am I missing something?  I do believe it is likely part of the API and I see it mentioned [here](https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/article_0110065209.html), I just think it is weird that it isn't in the help?

2. It looks like you have a stray /** on line 1046, can you remove that?

3. Can you add a comment explaining the DescendantorSelfNodesSelector?  I found it described here: https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/section_163170591936.html

brunogaleski commented:
Thanks @MrRob for the inputs.

1. When I created the charts, I couldn't find any documentation. I had to manually create one using the Analytics interface and then print it in the console to inspect the structure.
2. Removed.
3. I added the description, but it appears to be outdated, as I'm using it in a method that isn't listed there.
